modules/local/filter_genomes.nf

```diff
@@ -2,7 +2,7 @@ process FILTER_GENOMES {
     tag "$meta.id"
     label 'process_high'
 
-    conda "conda-forge::r-tidyverse=1.3.1 conda-forge::r-data.table=1.14.0 conda-forge::r-dtplyr=1.1.0"
+    conda "conda-forge::tar"
```

Comment:
How can you replace all those with `tar`?
Moreover, they have moved the conda part to some sort  of `environment.yml` file, I think. But maybe that comes later with a template update. And maybe that's only relevant for nf-core modules.

Reply:
this was a mistake: I will fix it.